dxr/query.py

```diff
@@ -97,13 +97,35 @@ def results(self, offset=0, limit=100):
                          ...]}
 
         """
+        enabled_filters_by_name = filters_by_name(self.enabled_plugins)
+
+        def group_filters_by_term(predicate):
+            """Return an iterable of a list of ES filters for each term, filtered on
```

Comment:
"an iterable of lists of" makes more sense. Or sometimes I say "Yield lists of…" for conciseness.